waterloop-site/src/components/SustainableTech/NavBar/SideBar.tsx

```diff
@@ -145,55 +141,73 @@ class Sidebar extends React.Component<MyProps> {
         </StyledToggle>
         <ListContainer className={this.props.sidebarOpen ? '' : 'closed'}>
           <div>
-            <IconBlack src={Waterlooplogo}></IconBlack>
-            <Link
-              to="/"
+            <ScrollLink
               onClick={(): void => {
                 this.props.handleClickSidebar(false);
+                setTimeout(function () {
+                  scrollTo('home-scroll');
+                }, 400);
               }}
             >
               Home
-            </Link>
+            </ScrollLink>
           </div>
           <div>
-            <Link
-              to="/the-flock"
+            <ScrollLink
               onClick={(): void => {
                 this.props.handleClickSidebar(false);
+                setTimeout(function () {
+                  scrollTo('IDlocation2');
+                }, 400);
               }}
             >
-              Flock
-            </Link>
+              About
+            </ScrollLink>
           </div>
           <div>
-            <Link
-              to="/team"
+            <ScrollLink
               onClick={(): void => {
                 this.props.handleClickSidebar(false);
+                setTimeout(function () {
+                  scrollTo('IDlocation3');
+                }, 400);
               }}
             >
               Team
-            </Link>
+            </ScrollLink>
+          </div>
+          <div>
+            <ScrollLink
+              onClick={(): void => {
+                this.props.handleClickSidebar(false);
+              }}
+            >
+              Schedule
+            </ScrollLink>
           </div>
           <div>
-            <Link
-              to="/sponsors"
+            <ScrollLink
               onClick={(): void => {
                 this.props.handleClickSidebar(false);
+                setTimeout(function () {
+                  scrollTo('IDlocation4');
+                }, 400);
               }}
             >
               Sponsors
-            </Link>
+            </ScrollLink>
           </div>
           <div>
-            <Link
-              to="/recruitment"
+            <ScrollLink
               onClick={(): void => {
                 this.props.handleClickSidebar(false);
+                setTimeout(function () {
+                  scrollTo('IDlocation5');
+                }, 400);
               }}
             >
-              Join Us
-            </Link>
+              FAQ
+            </ScrollLink>
           </div>
```

Comment:
Would you be able to reduce the amount of "repeated" code? For example, by using `map` on an array of objects like:
`[
  {
    id: "home-scroll",
    name: "Home"
  },
  { ... },
  { ... }
]`